CMakeLists.txt

```diff
@@ -204,12 +200,13 @@ if(ENABLE_WS)
       list(APPEND sources ${CMAKE_CURRENT_SOURCE_DIR}/src/wss_address.hpp
            ${CMAKE_CURRENT_SOURCE_DIR}/src/wss_address.cpp ${CMAKE_CURRENT_SOURCE_DIR}/src/wss_engine.hpp
            ${CMAKE_CURRENT_SOURCE_DIR}/src/wss_engine.cpp)
+      include_directories(${GNUTLS_INCLUDE_DIRS})
```

Comment:
in general I would prefer to change to modern cmake an use:

`target_include_directories(...)`